.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,10 @@
+---
+name: "\U0001F389 Feature request"
+about: Please use https://contributors.scala-lang.org/ for enhancement ideas
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Please use https://contributors.scala-lang.org/ for enhancement ideas instead of posting them to the issue tracker.
```

Comment:
shorter: "For enhancement ideas, please use https://contributors.scala-lang.org/, not scala/bug"